### PR TITLE
JP-3334 Commanded Guide Star postion telemetry is always relative to FGS1

### LINKED ISSUE #7802: Remove translation of between FGS1 and FGS2 in pointing algorithm
_Issue [JP-3334](https://jira.stsci.edu/browse/JP-3334) was created on JIRA by [Jonathan Eisenhamer](https://jira.stsci.edu/secure/ViewProfile.jspa?name=eisenham):_

It has been found that the pointing commanding system always uses FGS1 as the basis. However, the algorithm as specified in `set_telescope_pointing` translates to FGS2 when FGS2 is in use.

Remove this translation.

### PR DESCRIPTION
**Pull request by stscieisenhamer**
Resolves [JP-3334](https://jira.stsci.edu/browse/JP-3334)

Closes #7802 

This PR addresses an algorithmic problem within the telescope pointing determination. Previously, the commanded guide star position telemetry had presumed to be relative to the active FGS, either 1 or 2. However, that is not the case. Commanded guide star telemetry is always relative to FGS1. 

**Checklist for maintainers**
- [x] added entry in `CHANGES.rst` within the relevant release section
- [x] updated or added relevant tests
- [x] updated relevant documentation
- [x] added relevant milestone
- [x] added relevant label(s)
- [x] ran regression tests, post a link to the Jenkins job below.
      [How to run regression tests on a PR](https://github.com/spacetelescope/jwst/wiki/Running-Regression-Tests-Against-PR-Branches)
- [ ] Make sure the JIRA ticket is [resolved properly](https://github.com/spacetelescope/jwst/wiki/How-to-resolve-JIRA-issues)

### CODE REVIEW

**Comment by stscieisenhamer** [comment]
[Regression test](https://plwishmaster.stsci.edu:8081/job/RT/job/eisenhamer-dev/321/)
29 unrelated errors but unclear as to why these are different than the nightly run.

[Re-run](https://plwishmaster.stsci.edu:8081/job/RT/job/eisenhamer-dev/322/)

**Comment by stscieisenhamer** [comment]
And about the style check, I call shenanigans. `ruff` 0.0282 does not give this error AND not relevant to the PR.

File: jwst/lib/set_telescope_pointing.py [hunk 1]
```diff
@@ -2233,13 +2233,19 @@ def calc_v3pags(t_pars: TransformParameters):
     -------
     v3pags : float
         The V3 Position Angle at the Guide Star, in degrees
+
+    Notes
+    -----
+    Modification for release post-1.11.3: The commanded position of the guide
```
Comment (nden) [inline jwst/lib/set_telescope_pointing.py:2239]: `post-1.11.3` is a release of what?

**Review by nden — APPROVED** [review]
A minor comment from a bystander.

**Pull request merged**